sopel/plugins/__init__.py

```diff
@@ -80,6 +83,18 @@ def find_sopel_modules_plugins():
             yield handlers.PyModulePlugin(name, 'sopel_modules')
 
 
+def find_entrypoints_plugins(group='sopel.plugins'):
```

Comment:
Other comments go into why I'm suggesting this.

```suggestion
def find_entry_point_plugins(group='sopel.plugins'):
```